test/test_retriever.py

```diff
@@ -517,21 +525,20 @@ def test_getmd5_path():
     exp_hash = '0bec5bf6f93c547bc9c6774acaf85e1a'
     assert getmd5(data=data_file, data_type='file') == exp_hash
 
-
-def test_json2csv():
+@pytest.mark.parametrize("test_name, json_data, header_values, row_key, expected", json2csv_datasets)
+def test_json2csv(test_name, json_data, header_values, row_key, expected):
     """Test json2csv function.
 
     Creates a json file and tests the md5 sum calculation.
```

Comment:
@ashishpriyadarshiCIC - Are we still calculating md5? Maybe we can update the comment if not.